airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -190,9 +190,80 @@ definitions:
       schema:
         $ref: "#/schemas/email_subscriptions"
 
+  campaigns_stream:
+    $ref: "#/definitions/stream_base"
+    name: campaigns
+    primary_key:
+      - id
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /email/public/v1/campaigns
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - campaigns
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: offset
+        page_size_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: count
+        pagination_strategy:
+          type: CursorPagination
+          page_size: 500
+          cursor_value: "{{ response.get('offset') }}"
+          stop_condition: "{{ not response.get('hasMore') }}"
+      decoder:
+        type: JsonDecoder
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: lastUpdatedTime
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ format_datetime(config.get('start_date', '2006-06-01T00:00:00Z'), '%ms', '%Y-%m-%dT%H:%M:%SZ') }}"
```

Comment:
just to confirm my understanding: We receiving incoming `start_date` from the config as a ISO8601. But we need to convert it to millisecond format since that is the time format of the records returned from the API (to do the filter comparison). And we also store the last seen record in state as milliseconds? 

If so then I think this makes sense

Reply:
yep, you're right, it's transforming config or default start date to "%ms" format to have correct comparison